packages/baseclient/src/config/config.js

```diff
@@ -1,13 +1,16 @@
 const basePath = window.location.origin +
   window.location.pathname.match(/^\/[\w-]*\/?/)[0];
+const buildPath = window.location.origin +
+  window.location.pathname.match(/^(\/[\w-]*)*\/\/?/)[0]
 const shogun2Path = basePath + 'rest/applications/';
 let staticPath = basePath + 'resources/appContext.json';
 const appMode = typeof(APP_MODE) != "undefined" ? APP_MODE : undefined;
+const nodeEnv = typeof(process.env.NODE_ENV) != "undefined" ? process.env.NODE_ENV : undefined;
 
 export default {
   appContextPath: !appMode || appMode.indexOf('shogun2') > -1 ? shogun2Path : appMode.indexOf('static') > -1 ? staticPath : null,
   layerPath: basePath + 'rest/layers',
-  locale: basePath + 'resources/i18n/{{lng}}.json',
+  locale: !nodeEnv || nodeEnv.indexOf('development') > -1 ?  basePath + 'resources/i18n/{{lng}}.json': nodeEnv.indexOf('production') > -1 ? buildPath + 'resources/i18n/{{lng}}.json' : null,
```

Comment:
This is a very long line. Can we have it shorter? With some of the logic put before the export?

Reply:
Sure.